### ISSUE TITLE
Feature: factory reset smart card (PINs)

### ISSUE THREAD

**Issue by jans23** [Nitrokey/nitrokey-app#320]
This should allow to factory-reset Nitrokey Start, Pro and Storage.

**Comment by szszszsz** [comment]
It could be done with `libusb`/`libusbx` if these would be the only commands sent to smartcard. Using `libccid` or `OpenSC` is an option as well.

**Comment by alex-nitrokey** [comment]
I am not sure if this is a own issue, it seems related to me: 
it would be awesome to let users change user and admin PIN of Gnuk/NK Start devices as well. This would make it easier for users. They may could avoid the command line at all :)

**Comment by szszszsz** [comment]
Indeed, once the CCID interface will be handled then another devices/functionality could be managed as well.
Is it possible though to change the PIN without generating the keys?

**Comment by alex-nitrokey** [comment]
> Is it possible though to change the PIN without generating the keys?

No it isn't, but one can create keys within Thunderbird's Enigmail and change PINs afterwards.... Oh wait. You can change the PINs there anyway. Well, then it is a smaller group of people who would benefit :thinking:
